package.json

```diff
@@ -87,6 +88,7 @@
     "cid-tool": "~0.1.0",
     "cids": "~0.5.3",
     "debug": "^3.1.0",
+    "electron-webrtc": "~0.3.0",
```

Comment:
This is deliberately not installed (as it is not needed)

Reply:
any background on this? to better understand ?

Reply:
https://github.com/ipfs/js-ipfs#is-there-webrtc-support-for-js-ipfs-with-nodejs